core/install.js

```diff
@@ -28,7 +28,7 @@ exports.requireOrInstall = function(req, name) {
 	try {
 		return req(name);
 	}
-	catch(e) {
+	catch (e) {
 		if (!e || !e.code || e.code !== 'MODULE_NOT_FOUND') {
 			throw e;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2554993052/issues/source?bid=3282987&fileBranchId=3284644#l33)